.storybook/stories/Loading/Loading.tsx

```diff
@@ -18,6 +21,7 @@ export const Loading: FC<LoadingProps> = ({
         color={theme.colors[color]}
         style={styles.scale}
       />
+      {text && <Typography variant={'body-large'} text={text} center />}
```

Comment:
```suggestion
      {text && <Typography variant="body-large" text={text} center />}
```
Maybe `header-large/medium`